src/client/components/PostInfo/index.js

```diff
@@ -5,7 +5,7 @@ class PostInfo extends React.Component {
     return (
       <div className="post-info">
         <span className="likes">{this.props.postInfo.numOfLikes} likes</span>
-        <span>{this.props.postInfo.numOfComments} comments</span>
+        <span>{this.props.postInfo.comments.length} comments</span>
```

Comment:
this should be calculated on the backend side since in some cases we wont send back all of the comments